src/internal_rules.jl

```diff
@@ -672,6 +664,30 @@ function EnzymeRules.forward(::Const{typeof(cholesky)}, RT::Type, A; kwargs...)
     end
 end
 
+function _cholesky_forward(C::Cholesky, Σdot)
+    # Computes the cholesky forward mode update rule
+    # C.f. eq. 8 in https://arxiv.org/pdf/1602.07527.pdf
+    if C.uplo == 'U'
+        U = C.U
+        Udot = Σdot / U
+        ldiv!(U', Udot)
+        idx = diagind(Udot)
+        Udot[idx] ./= 2
+        triu!(Udot)
+        rmul!(Udot, U)
+        return Cholesky(Udot, 'U', 0)
```

Comment:
Notational consistency. Also, we should probably keep the same `.info` as the primal.
```suggestion
        U̇ = Ȧ / U
        ldiv!(U', U̇)
        idx = diagind(U̇)
        U̇[idx] ./= 2
        triu!(U̇)
        rmul!(U̇, U)
        return Cholesky(U̇, 'U', C.info)
```